drs_downloader_to_getm.wdl

```diff
@@ -159,11 +160,19 @@ task download {
             start_time=`date +%s`
             signed_urls=($(cat ~{manifest} | jq -r '.[] .url'))
             for signed_url in ${signed_urls[@]}; do
-                # this is going to create some crazy truncated names but it shouldn't make a difference in run times
-                wget ${signed_url} -P ${TMP_DL_DIR}/
+                wget ${signed_url} --output-document=${TMP_DL_DIR}/$(basename $signed_url)
             done
             end_time=`date +%s`
             total_time="$(($end_time-$start_time))"
+
+            # time just the md5sums
+            for signed_url in ${signed_urls[@]}; do
+                md5sum ${TMP_DL_DIR}/$(basename $signed_url)
+            done
+            end_time=`date +%s`
+            total_time_incl_md5="$(($end_time-$start_time))"
+            echo "~{downloader} ${total_time} seconds" > "~{downloader}.txt"
+            echo "~{downloader}+md5sum ${total_time_incl_md5} seconds" >> "~{downloader}.txt"
```

Comment:
This is my bad ... I proposed `~{downloader}+md5sum`, yet it turns out that`+` is not a valid character in Terra data table attribute names. Only alphanumeric characters, underscores, and dashes are allowed in attribute names, so I now propose: `~{downloader}_md5sum`